src/modules/languages/python.nix

```diff
@@ -176,10 +190,10 @@ in
     venv.enable = lib.mkEnableOption "Python virtual environment";
 
     venv.requirements = lib.mkOption {
-      type = lib.types.nullOr (lib.types.either lib.types.lines lib.types.path);
```

Comment:
Could we instead do `nullOr path`? 

This breaks backwards compatibility where previously there would be a list of requirements written as string and now it's a path as string.

Is there a reason we can't support `lines` by writing to a file?

Reply:
We can't use a path, because the file referenced by a path gets put into the store, and we lose the original name/path of the file in the user's environment.